flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/constraints/UniqueConstraint.java

```diff
@@ -1,111 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.flink.table.api.constraints;
-
-import org.apache.flink.annotation.PublicEvolving;
-import org.apache.flink.table.catalog.ResolvedSchema;
-
-import java.util.List;
-import java.util.Objects;
-
-import static org.apache.flink.util.Preconditions.checkNotNull;
-
-/**
- * A unique key constraint. It can be declared also as a PRIMARY KEY.
- *
- * @see ConstraintType
- * @deprecated See {@link ResolvedSchema} and {@link
- *     org.apache.flink.table.catalog.UniqueConstraint}.
- */
-@Deprecated
-@PublicEvolving
```

Comment:
I'm afraid we can't directly remove this `@PublicEvolving`  API. CC @luoyuxia

Reply:
+1. Share same concern. We must be cautious to dropping  `@PublicEvolving` api.

Reply:
> +1. Share same concern. We must be cautious about dropping `@PublicEvolving` api.

Thank you so much for review.  I will rollback '@ PublicEvolving' api Delete operation @luoyuxia @Aitozi